ImperatorToCk3/Source/Imperator/Families/Family.h

```diff
@@ -0,0 +1,33 @@
+#ifndef IMPERATOR_FAMILY_H
+#define IMPERATOR_FAMILY_H
+#include "newParser.h"
+
+namespace ImperatorWorld
+{
+class Family: commonItems::parser
+{
+  public:
+	Family(std::istream& theStream, int theFamilyID);
+
+	void updateFamily(std::istream& theStream);
```

Comment:
Please add tests demonstrating the use of updateFamily(). Tests serve not just as a demonstration of the correctness of code, but as documentation and examples of the interfaces.

Reply:
It's used by Families parser when a family's ID appears more than once. I've added this test to FamiliesTests to check if it the values are updated:
```
TEST(ImperatorWorld_FamiliesTests, familiesCanBeLoaded)
{
	std::stringstream input;
	input << "=\n";
	input << "{\n";
	input << "42={}\n";
	input << "43={}\n";
	input << "}";

	const ImperatorWorld::Families families(input);
	const auto& familyItr = families.getFamilies().find(42);
	const auto& familyItr2 = families.getFamilies().find(43);

	ASSERT_EQ(42, familyItr->first);
	ASSERT_EQ(42, familyItr->second->getID());
	ASSERT_EQ(43, familyItr2->first);
	ASSERT_EQ(43, familyItr2->second->getID());
}
```
Is it enough?

Reply:
Perfect